cli/src/torii/routing.rs

```diff
@@ -319,8 +333,10 @@ pub(crate) async fn handle_rejection(rejection: Rejection) -> Result<Response, R
         TxTooBig => return unhandled(rejection),
         Config(_err) => return unhandled(rejection),
         PushIntoQueue(_err) => return unhandled(rejection),
-        Status(_err) => return unhandled(rejection),
         ConfigurationReload(_err) => return unhandled(rejection),
+        #[cfg(feature = "telemetry")]
+        Status(_err) => return unhandled(rejection),
+        #[cfg(feature = "telemetry")]
         Prometheus(_err) => return unhandled(rejection),
```

Comment:
just a side note, how does the conditional compilation affect enum layout and subsequently enum serialization (e.g. for parity scale)? Is this something to consider? What do you think?

Reply:
Created #1976. Should discuss there